src/ripple/overlay/PeerSet.h

```diff
@@ -86,6 +85,11 @@ class PeerSetBuilder
     build() = 0;
 };
 
+/**
+ * This constructor requires an Application because sometimes an Overlay is
+ * not built, and even when it is, it is built after the objects that want
+ * PeerSets. It is non-const because it may construct a new Journal.
+ */
 std::unique_ptr<PeerSetBuilder>
 make_PeerSetBuilder(Application& app);
```

Comment:
`PeerSetBuilder` was added as part of [FLR1](https://github.com/XRPLF/rippled/pull/3659). It is used only by `LedgerReplayer`. It could be replaced with calls to a new function that takes an `Application&` and returns a `std::unique_ptr<PeerSet>`, but that is out-of-scope for this PR.